include/deal.II/fe/fe_nothing.h

```diff
@@ -82,10 +82,12 @@ class FE_Nothing : public FiniteElement<dim,spacedim>
 public:
 
   /**
-   * Constructor. Argument denotes the number of components to give this
-   * finite element (default = 1).
+   * Constructor. First argument denotes the number of components to give this
+   * finite element (default = 1). Second argument decides whether FE_Nothing
+   * will dominate any other FE in compare_for_face_domination() (default = false).
    */
-  FE_Nothing (const unsigned int n_components = 1);
+  FE_Nothing (const unsigned int n_components = 1,
+              const bool dominate = false);
```

Comment:
yes, that's my intention.

btw, note that cases like `FESystem(FE_Q(1),1,FE_Nothing,1)` vs `FESystem(FE_Q(2),1,FE_Q(1),1)` will already result in constraining Q1 to be come zero at the interface as the first FE will dominate.